agents/omnibusproject/README.md

```diff
@@ -0,0 +1,90 @@
+# The Omnibus Project
+
+This is an Agent that answers questions about the hit podcast, 
+[The Omnibus Project](https://www.omnibusproject.com/), by Ken Jennings and John Roderick.
+This podcast is a compendium of arcane trivia and esoteric knowledge, and is a perfect
+demonstration use case for Fixie agents being able to take what is a complex,
+meandering dialogue between two people and turn it into a useful knowledge base.
+
+The agent works by downloading the audio for each episode, transcribing it using the
+Deepgram speech-to-text AI model, and creating a Fixie agent using the resulting
+transcripts.
+
+You can try it yourself here: https://app.fixie.ai/agents/mdw/omnibusproject
+
+Try asking questions like:
+
+* Why does John pronounce "England" as "Englang"?
+* Summarize the episode about Rhinos in Europe
+* Who was Magic Alex?
+
+If you want to run the agent yourself, here's how.
+
+## Downloading and transcribing the podcast audio
+
+The code in `downloader` deals with downloading and transcribing the audio for all of the
+Omnibus Project episodes. It checks the RSS feed for the podcast, downloads any new episodes, 
+uses the Deepgram speech-to-text model to transcribe the audio, and stores the results in a
+Google Cloud Storage bucket.
+
+To use this tool, you need to do the following.
+
+First, create a Google Cloud Storage project and GCS bucket. Edit `downloader/main.py` and
+change the `GCS_BUCKET` variable to the name of your bucket.
+
+For now, the contents of the bucket need to be world-readable for Fixie to be able to fetch
+the transcripts and answer questions. You can do this by running:
+
+```
+gsutil defacl set public-read gs://<your bucket name>
```

Comment:
```suggestion
$ gsutil defacl set public-read gs://<your bucket name>
```

The other command line code blocks have a leading `$` so we should include it here for consistency.